### ISSUE TITLE
Option to ignore specified items

### ISSUE THREAD

**Issue by tunapuff** [VPenkov/okayNav#6]
Great plugin, we've been looking for something like this at work. Would it be possible to add an option where you could specify menu items that are excluded from being moved into the collapsed navigation e.g. I want to have the "Contacts" link in your demo to always remain visible.

**Comment by VPenkov** [comment]
Yes, that's on the roadmap. Should be a few weeks before I get to it.
You'll be able to assign priority or exclude items altogether via data attributes.

**Comment by tunapuff** [comment]
Perfect, thank you so much!

**Comment by berend-de-jong** [comment]
Looking forward to this feature. It's rather vital to keep priority items visible, even if they normally appear on the right side (Contact).

**Comment by AndrewIngram** [comment]
What about something like:

`<li data-okaynav-priority="1" />`

**Comment by VPenkov** [comment]
@AndrewIngram yes, that's pretty much what I meant.

**Comment by Yearhouse** [comment]
Hi Penkov.

Do you have an idea of when this will be released? Looking very much forward to this feature.
Thanx for a great plugin :)
